pkg/controllers/workgenerator/controller_integration_test.go

```diff
@@ -146,6 +146,24 @@ var _ = Describe("Test Work Generator Controller", func() {
 			// flip the binding state to bound and check the work is created
 			binding.Spec.State = placementv1beta1.BindingStateBound
 			Expect(k8sClient.Update(ctx, binding)).Should(Succeed())
+			Eventually(func() error {
+				// Eventually update the binding with for RolloutStarted condition in case it runs into a conflict error
+				if err := k8sClient.Get(ctx, types.NamespacedName{Name: binding.Name}, binding); err != nil {
+					return err
```

Comment:
is there a way to not copy paste this code in every case? ie. put this in the create binging loigc.